apps/managedidentity/docs/RUN_IMDS.md

```diff
@@ -0,0 +1,162 @@
+# Running the IMDS Source
+
+This file will talk you through the steps required to run the IMDS Managed Identity Source locally
+
+## Files to Modify
+
+Firstly, to run IMDS locally, we need to change some files.
+
+1. Comment out everything in `client_certificate_sample.go` besides `package main`
+2. In `sample_utils.go`, replace the Config struct with this
+
+```
+type Config struct {
```

Comment:
A standalone sample for MSI where this is file is preformatted would be better than docs.

Reply:
Modified the existing code so we use our managedidentity_sample.go